query_processor/services/content_router.py

```
@@ -0,0 +1,113 @@
+            )
+            
+            # Route to appropriate processor
+            if request_type in ['video', 'playlist']:
+                return await ContentRouterService._route_video_request(query_request)
+            elif request_type == 'topic':
+                return await ContentRouterService._route_topic_request(query_request)
+            else:
+                raise ValueError(f"Unsupported request type: {request_type}")
+                
+        except ValidationError as e:
+            logger.warning(f"Content validation failed for {request_type}: {str(e)}")
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)